src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -118,6 +118,15 @@ def configure_container_dict(
     host_server_info_file : Path
     remove_server_info_file: bool
 
+    Raises
+    ------
+    KeyError
+        If license server is not specified through an environment variable or in ``container_dict``.
+    ValueError
+        If server info file is specified through both a command-line argument inside ``container_dict`` and the  ``container_server_info_file`` parameter.
+    ValueError
+        If ``fluent_image`` or ``image_tag`` and ``image_name`` are not specified.
```

Comment:
The bonus value here is that developers can see where we need to design higher-level exception types.

See #2098

@mkundu1 @raph-luc

Reply:
I think that adding higher-level custom exception classes is something that may not end up being helpful, particularly if we aren't catching those custom exceptions and also don't expect users to catch and handle these exceptions in different ways necessarily, if that makes sense. Will comment on the issue tracker